packit/cli/utils.py

```diff
@@ -135,41 +135,28 @@ def get_packit_api(
     lp_upstream = None
     lp_downstream = None
 
-    upstream_compared = False
-
     for url in remote_urls:
         remote_hostname = get_hostname_or_none(url=url)
         if not remote_hostname:
             continue
 
         if upstream_hostname:
-            upstream_compared = True
             if remote_hostname == upstream_hostname:
                 lp_upstream = local_project
                 logger.info("Input directory is an upstream repository.")
                 break
 
-        if (
-            package_config.dist_git_base_url
-            and remote_hostname in package_config.dist_git_base_url
+        if package_config.dist_git_base_url and (
+            remote_hostname in package_config.dist_git_base_url
+            or remote_hostname in "pkgs.fedoraproject.org"
```

Comment:
```suggestion
            or remote_hostname in "src.fedoraproject.org"
```
or

```suggestion
            or remote_hostname == "src.fedoraproject.org"
```

What about some constant list? (So we can support staging instance as well.)

Reply:
yeah, I can move it to constants, but here I really mean pkgs.fp.p, not src.fp.o
```
origin  ssh://ttomecek@pkgs.fedoraproject.org/forks/ttomecek/rpms/python-docker.git (fetch)
origin  ssh://ttomecek@pkgs.fedoraproject.org/forks/ttomecek/rpms/python-docker.git (push)
upstream        ssh://ttomecek@pkgs.fedoraproject.org/rpms/python-docker (fetch)
upstream        ssh://ttomecek@pkgs.fedoraproject.org/rpms/python-docker (push)
```

Reply:
OK, so maybe both of them for people using Pagure+git instead of fedpkg?

![Snímek z 2019-10-09 10-15-23](https://user-images.githubusercontent.com/20214043/66463623-bfba9b00-ea7d-11e9-90ce-d8a398c696db.png)